tests/test_trnsys.py

```diff
@@ -23,6 +32,48 @@ def step_forward(self, steps: int) -> (bool, int):
             return (False, 1)
         return (True, 0)
 
+    def get_output_value(self, unit: int, output_number: int) -> (float, int):
+        if unit not in self.units:
+            return (math.nan, 1)
+        key = f"{unit}:{output_number}"
+        value = self.unit_outputs.get(key)
+        if value is None:
+            return (math.nan, 2)  # output is not available for this unit
```

Comment:
Same nit about inline comments. I'd either put it above, or rename a variable to make it more clear. Like, could value be renamed to output?

Reply:
Some inline comments removed in 0bf6fba.

The other ones I think are an indication that I was too cavalier with the `MockTrnsysLib` and the extra methods I gave it to set the mocked simulation's state.  Let's pair on this and be more thoughtful with how we manage and manipulate its state.